src/Sulu/Bundle/LocationBundle/Geolocator/Service/NominatimGeolocator.php

```diff
@@ -72,7 +76,7 @@ public function locate(string $query): GeolocatorResponse
             );
         }
 
-        $results = \json_decode($response->getBody(), true);
+        $results = $response->toArray();
```

Comment:
we need here handle if its `Guzzle` or `Symfony` client because of our bc promise.